src/main/java/core/basesyntax/dao/StorageDao.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.dao;
+
+import java.util.Map;
+
+public interface StorageDao {
+    void update(String fruit, Integer quantity);
+
+    Integer getCurrentFruits(String fruit);
```

Comment:
You return not fruit, you return the quantity of some fruit, so rename this method